roles/openshift_gcp/tasks/setup_scale_group_facts.yml

```diff
@@ -31,7 +31,7 @@
   add_host:
     name: "{{ hostvars[item].gce_name }}"
     groups: nodes, new_nodes
-    openshift_node_bootstrap: False
+    openshift_node_bootstrap: "{{ openshift_node_bootstrap | default(True) }}"
```

Comment:
Allows older GCP nodes to have their bootstrap behavior inferred from the global (i.e. a node which was not bootstrapped in 3.9 defaults to being bootstrapped unless a user asks for static configuration).